quic/client/QuicClientTransport.cpp

```diff
@@ -1289,10 +1276,6 @@ void QuicClientTransport::setHostname(const std::string& hostname) {
   hostname_ = hostname;
 }
 
-void QuicClientTransport::setPskCache(std::shared_ptr<QuicPskCache> pskCache) {
-  pskCache_ = std::move(pskCache);
-}
-
```

Comment:
Let me know if I missed another API that's introduced in this diff or some other diffs as a replacement. Otherwise, can you kep the setPskCache API in the transport? We have other libraries need to set the PSK. I guess you can passes the pskCache on to the handshakeLayer?

Reply:
Yes, absolutely. You should set the psk cache when building the FizzClientQuicHandshakeContext that you pass to the transport.